.github/CODEOWNERS

```diff
@@ -12,64 +12,90 @@
 
-# Each line in this file is a file pattern followed by one or more
-# @github_usernames and/or e-mail addresses corresponding to users with write
-# permissions for the repository.
+# More info:
 # https://help.github.com/en/articles/about-code-owners
 
 # Order is important; the user(s) in the last matching pattern here for a file will
 # be that file's owner(s).
 
-# The users below will be the default owners for everything in the repository.
 # Unless a line further in this file matches, all of the users here will be
 # requested for review when someone opens a pull request.
-*	@davetcoleman @v4hn @rhaschke 
 
-# The lines below correspond to directories with a package.xml file and the
-# maintainers specified in that file
-
-/moveit_plugins/moveit_ros_control_interface/	@ipa-mdl @bmagyar
-
-/moveit_plugins/moveit_fake_controller_manager/	@v4hn @rhaschke
-
-/moveit_plugins/moveit_simple_controller_manager/	@mikeferguson @v4hn
-
-/moveit_plugins/moveit_controller_manager_example/	@v4hn
-
-/moveit_core/	@davetcoleman @rhaschke @v4hn @mlautman
-
-/moveit_commander/	@v4hn @rhaschke @willcbaker
-
-/moveit/	@130s
-
-/moveit_kinematics/	@rhaschke @gavanderhoorn @jrgnicho
-
-/moveit_experimental/	@AndyZe
-
-/moveit_ros/perception/	@mikeferguson @jonbinney
-
-/moveit_ros/manipulation/	@v4hn @felixvd 
-
-/moveit_ros/benchmarks/	@davetcoleman @MohmadAyman 
-
-/moveit_ros/planning_interface/	@mintar @rhaschke 
-
-/moveit_ros/robot_interaction/	@mikeferguson @rhaschke
-
-/moveit_ros/planning/	@henningkayser @v4hn @rhaschke 
-
-/moveit_ros/warehouse/	@mikeferguson @dg-shadow 
-
-/moveit_ros/move_group/	@rhaschke @IanTheEngineer @v4hn
-
-/moveit_ros/visualization/	@rhaschke @jonbinney @christian-rauch 
-
-/moveit_setup_assistant/	@davetcoleman @rhaschke @MohmadAyman 
-
-/moveit_planners/ompl/	@BryceStevenWilley @zkingston
-
-/moveit_planners/chomp/chomp_interface/ @raghavendersahdev @knorth55 @bmagyar
-
-/moveit_planners/chomp/chomp_optimizer_adapter/	@raghavendersahdev @knorth55 @bmagyar
-
-/moveit_planners/chomp/chomp_motion_planner/	@raghavendersahdev @knorth55 @bmagyar
+# DEFAULT REVIEWERS:
+
+*                                                           @v4hn @rhaschke @davetcoleman
+
+# FILE-SPECIFIC REVIEWERS:
+
+/.github/CODEOWNERS                                         @nbbrooks @davetcoleman
+/README.md                                                  @davetcoleman
+
+# FOLDER-SPECIFIC REVIEWERS:
+
+/moveit_plugins/moveit_ros_control_interface/	            @ipa-mdl @bmagyar
+/moveit_plugins/moveit_fake_controller_manager/	            @v4hn @rhaschke
+/moveit_plugins/moveit_simple_controller_manager/	    @mikeferguson @v4hn
+/moveit_plugins/moveit_controller_manager_example/	    @v4hn
+
+/moveit_core/background_processing/                         @mlautman
+/moveit_core/backtrace/                                     @mlautman
+/moveit_core/collision_detection/                           @rhaschke
```

Comment:
I'm not particularly experienced in collision checking. Do we have a true expert for this?

Reply:
I wouldn't call myself an expert, but I'm familiar with the the collision checking stack.  I'd be willing to to review changes there.